fvm/env.go

```diff
@@ -535,12 +535,16 @@ func (e *hostEnv) Logs() []string {
 	return e.logs
 }
 
-func (e *hostEnv) Hash(data []byte, hashAlgorithm runtime.HashAlgorithm) ([]byte, error) {
+func (e *hostEnv) Hash(data []byte, tag string, hashAlgorithm runtime.HashAlgorithm) ([]byte, error) {
 	if e.isTraceable() {
 		sp := e.ctx.Tracer.StartSpanFromParent(e.transactionEnv.traceSpan, trace.FVMEnvHash)
 		defer sp.Finish()
 	}
 
+	if len(tag) > 0 {
+		return nil, fmt.Errorf("specifying the tag when computing a hash is not yet suported")
+	}
+
 	hashAlgo := RuntimeToCryptoHashingAlgorithm(hashAlgorithm)
```

Comment:
This works here as long as `RuntimeToCryptoHashingAlgorithm` doesn't support `KMAC`.